examples/binaries/new-meta/state-v2/src/wasm.rs

```diff
@@ -3,14 +3,16 @@ use gmeta::metawasm;
 use gstd::prelude::*;
 
 #[metawasm]
-pub trait Metawasm {
-    type State = Vec<Wallet>;
+pub mod metafuncs {
+    pub type State = Vec<Wallet>;
 
-    fn wallet_by_id(id: Id, state: Self::State) -> Option<Wallet> {
+    /// Returns a wallet with the given `id`.
+    pub fn wallet_by_id(state: State, id: Id) -> Option<Wallet> {
```

Comment:
Haven't we agreed to demonstrate functions reusing (call one function from another or something similar), aren't we?

Reply:
same agreement on demo of multiple arguments